README.md

```diff
@@ -96,66 +96,74 @@ in current directory:
     $> python -m duecredit examples/example_scipy.py
     I: Simulating 4 blobs
     I: Done clustering 4 blobs
+    
     DueCredit Report:
-    - scipy (v 0.14.1) [1]
-      - scipy.cluster.hierarchy:linkage (Single linkage hierarchical clustering) [2]
-    - numpy (v 1.8.2) [3]
-
-    2 modules cited
-    1 functions cited
-
+    - Scientific tools library / scipy (v 0.14) [1]
+      - Single linkage hierarchical clustering / scipy.cluster.hierarchy:linkage (v 0.14) [2]
+    
+    1 package cited
+    0 modules cited
+    1 function cited
```

Comment:
mmmh, I'm not really enthusiastic about it ;p isn't the whole idea of tags to reduce the noise? we could have another command like `duecredit stats` that provides some stats?